ppr-ui/src/components/userAccess/ReviewConfirm/ListRequirements.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <ol class="pl-0">
+    <div
+      v-for="(requirement, index) in requirements"
+      :key="index"
+    >
+      <li class="ml-14 px-3">
+
+        <!-- Requirement with tooltip text -->
+        <p  v-if="requirement.tooltipText" class="ma-0">
+          <b>
+            {{ requirement.boldTextPreTooltip }}
+            <v-tooltip top content-class="top-tooltip" transition="fade-transition">
+              <template #activator="{ on }">
+                <span v-on="on" class="dotted-underline" tabindex="0">
+                  {{ requirement.underlinedText }}
+                </span>
+              </template>
+              <div class="py-2">{{ requirement.tooltipText }}</div>
+            </v-tooltip>
+            {{ requirement.boldTextPostTooltip }}
+          </b>
+            {{ requirement.regularText }}
+        </p>
+
+        <!-- Requirement without tooltip text -->
+        <p class="ma-0" v-else>
+          <b>{{ requirement.boldText }}</b>
+          {{ requirement.regularText }}
+        </p>
+      </li>
+      <v-divider v-if="index + 1 !== requirements.length" class="my-7 ml-0 mr-1" />
+    </div>
+  </ol>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent } from 'vue-demi'
+import { useStore } from '@/store/store'
+import { storeToRefs } from 'pinia'
+import { RequirementsConfigIF } from '@/interfaces'
+import { userAccessRequirements } from '@/resources'
+
+export default defineComponent({
+  name: 'listRequirements',
+  setup () {
+    const { getMhrSubProduct } = storeToRefs(useStore())
+
+    const requirements = computed((): RequirementsConfigIF[] => userAccessRequirements[getMhrSubProduct.value])
+
+    return { requirements }
```

Comment:
Not a big deal but the only thing separating this component from being completely reusable for any list formatted like this, would be these bindings. 

You could* pass this in as a prop and you'd pretty purely atomic, your call though!

Reply:
Will change it to a prop.

Reply:
I changed it to a prop but I still kept the prop called `requirements`, this can defiantly be made more abstract, but it just to hint to future developers, that they might need to make some minor changes before it can be reused, because I haven't tested or made sure that it is reusable out the box.

Reply:
Absolutely sounds good. 
Such is always the case, don't need to make everything full proof / future proof. 

That can always be done incrementally and iteratively.